unittests/test_test_runner.py

```diff
@@ -161,7 +161,7 @@ def test_run_jobs_get_task(self, mock_validate_response, mock_client):
         mock_validate_response.return_value = {}
         mock_client.return_value = MagicMock()
 
-        test_runner = TestRunner("test", "test", "v1.0")
+        test_runner = TestRunner("TES", "test", "1.0.0")
```

Comment:
I had to add TES here since our services enum only supports TES. This will be removed later when the runner is separated.

Reply:
Yeah, sure, that's fine